module/plugins/hoster/LibGenComics.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+import re
+from module.network.HTTPRequest import BadHeader
+
+from .Http import Http
+
+class LibGenComics(Http):
+    __name__ = "LibGenComics"
+    __type__ = "hoster"
+    __version__ = "0.10"
+    __status__ = "testing"
+
+    __pattern__ = r'https?://libgen.io/comics0/.+\.(cbr|cbz|zip|rar|pdf)'
+    __config__ = [("activated", "bool", "Activated", True)]
+
+    __description__ = """Download comics from libgen.io respecting throttling limits"""
+    __license__ = "GPLv3"
+    __authors__ = [("Yann Jouanique", "yann.jouanique@gmail.com")]
+
+    def setup(self):
```

Comment:
Please remove `multi_dl` and `limit_dl`, they are not used.
```
   def setup(self):
        self.chunk_limit = -1
        self.resume_download = True
        self.multiDL = False
        self.limitDL = 1
```

Reply:
`multiDL` is `False` so `limitDL` is unnecessary.
```
   def setup(self):
        self.chunk_limit = -1
        self.resume_download = True
        self.multiDL = False
```

Reply:
OK, I wasn't entirely sure of the usage but indeed looks like `multiDL` is enough.